dockstore-client/src/main/java/io/dockstore/client/cli/nested/WesCommandParser.java

```diff
@@ -60,6 +60,8 @@ public boolean isHelp() {
     public static class CommandLaunch extends WesMain {
         @Parameter(names = "--entry", description = "Complete workflow path in Dockstore (ex. NCI-GDC/gdc-dnaseq-cwl/GDC_DNASeq:master)", required = true)
         private String entry;
+        @Parameter(names = "--provision-locally", description = "Indicates that the provided entry should be downloaded locally and pass inline to the WES server.")
```

Comment:
For the CLI, I associate "provision" with provisioning files (our file provisioning plugins). so I feel the word is getting a bit overloaded. 

Also, the key point here is the request to WES; one request has a reference to the descriptor, the other request has the descriptors inline. How we create an inline request (by downloading files) is arguably an implementation detail.

So, how about instead of `--provision-locally`, something like `--inline-descriptors` or `--inline-workflow`, and the description only talks about how the request is formed?

This is a proposal I'm not sure of, and willing to be talked out of.